@xen-orchestra/web-core/lib/components/state-hero/VtsStateHero.vue

```diff
@@ -113,5 +122,24 @@ const imageSrc = computed(() => {
       width: 80%;
     }
   }
+
+  &.table {
+    padding: 5rem;
```

Comment:
To match the design system:
```suggestion
    padding: 4rem;
```